src/Storages/PartitionCommands.cpp

```diff
@@ -97,4 +99,86 @@ std::optional<PartitionCommand> PartitionCommand::parse(const ASTAlterCommand *
         return {};
 }
 
+std::string PartitionCommand::typeToString() const
```

Comment:
Maybe we can use this method later in `ASTAlterCommand::formatImpl`

Reply:
Probably we can, but it require a lot of changes and I'm not sure that code will became much simplier.